src/genotyper.cpp

```diff
@@ -81,21 +80,26 @@ void Genotyper::run(AugmentedGraph& augmented_graph,
 
     // Find all the snarls in either the main graph or the subset, and put them
     // in this SnarlManager
-    SnarlManager manager;
+    SnarlManager manager = CactusSnarlFinder(graph, ref_path_name).find_snarls();
 
-    // We need to decide if we want to work on the full graph or just on the subgraph that has any support.
-    // We make the subset a local out here so it will stick around as long as we
-    // use the SnarlManager.
-    VG subset;
-    if (subset_graph) {
-        subset = make_subset_graph(graph, ref_path_name, reads_by_name); 
-        manager = subset.paths.has_path(ref_path_name) ?
-            CactusSnarlFinder(subset, ref_path_name).find_snarls()
-            : CactusSnarlFinder(subset).find_snarls();
-    } else {
-        algorithms::sort(&graph);
-        manager = CactusSnarlFinder(graph, ref_path_name).find_snarls();
-    }
+    // TODO: I've removed this option because it has the chance to create incorrect Snarls from the subset
+    // graph that are not actually snarls in the full graph. If we are going to retain this feature, we need
+    // to use the subset graph throughout the genotyping process, but that's currently not possible because
+    // the rest of the code uses an AugmentedGraph instead of a VG.
+    
+//    // We need to decide if we want to work on the full graph or just on the subgraph that has any support.
+//    // We make the subset a local out here so it will stick around as long as we
+//    // use the SnarlManager.
+//    VG subset;
```

Comment:
Shouldn't we just drop all this code instead of commenting it out?

Reply:
I thought about that, but I figured that if someone came back in the future and wanted to re-implement this feature it might be handy to have a prototype to look at without digging into the git history. If you feel strongly about it, I can remove this.